packages/bng-metric-service/src/helpers/extractors/__tests__/bng-metric-single-data-extractor.spec.js

```diff
@@ -17,7 +17,7 @@ describe('BNG data extractor test', () => {
     const response = await bngMetricDataExtractor.extractContent(readableStreamv4, options)
 
     expect(response.d1OffSiteHabitatBaseline.length).toEqual(5)
-    expect(Object.keys(response.d1OffSiteHabitatBaseline[0]).length).toEqual(5)
+    expect(Object.keys(response.d1OffSiteHabitatBaseline[0]).length).toEqual(6)
```

Comment:
As well as adding in the extra field to the count can you also check the data for the new field in the row being tested too please?